src/Monitoring/Monitoring.ArcadeServices/dashboard/arcade-services/quota.dashboard.json

```diff
@@ -197,7 +197,7 @@
             "dimension": "none",
             "metricName": "select",
             "rawQuery": true,
-            "rawQueryString": "customEvents\r\n| where $__timeFilter(timestamp)\n| where name == 'AzureResourceUsage'\r\n| project\r\n    timestamp,\r\n    resource = tostring(customDimensions.name),\r\n    location = tostring(customDimensions.location),\r\n    current = toreal(customMeasurements.current),\r\n    ['limit'] = toreal(customMeasurements.['limit'])\r\n| where resource != 'NetworkWatchers'\n| where location == 'westus'\n| extend ['percent of limit'] = iff(['limit'] == 0, 0.0, current/['limit'] * 100)\n| summarize ['percent of limit'] = max(['percent of limit']) by bin(timestamp, $__interval), resource\r\n| order by timestamp asc, resource",
+            "rawQueryString": "customEvents\r\n| where $__timeFilter(timestamp)\n| where name == 'AzureResourceUsage'\r\n| project\r\n    timestamp,\r\n    resource = tostring(customDimensions.name),\r\n    location = tostring(customDimensions.location),\r\n    current = toreal(customMeasurements.current),\n    subscription = tostring(customDimensions.subscription),\r\n    ['limit'] = toreal(customMeasurements.['limit'])\n| where subscription in ($HelixSubscriptionId)\r\r\n| where resource != 'NetworkWatchers'\n| where location == 'westus'\n| extend ['percent of limit'] = iff(['limit'] == 0, 0.0, current/['limit'] * 100)\n| summarize ['percent of limit'] = max(['percent of limit']) by bin(timestamp, $__interval), resource\r\n| order by timestamp asc, resource",
```

Comment:
The grafana docs say queries for alerting cannot use dashboard variables ([source](https://grafana.com/docs/grafana/v6.7/alerting/create-alerts/#conditions)). Are you sure this works?